v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/utils/InputRecordProcessor.java

```diff
@@ -49,14 +54,16 @@ public static void processRecord(
       JSONObject newValuesJson = new JSONObject(newValueJsonStr);
       JSONObject keysJson = new JSONObject(keysJsonStr);
 
-      String dmlStatement =
-          DMLGenerator.getDMLStatement(
+      DMLGeneratorRequest dmlGeneratorRequest =
```

Comment:
Was any impact on worker memory observed during load test with creation of this new object for every request?